### PR TITLE
sync game-time to real-time & update worldLight

### PR DESCRIPTION
**Pull request by Sundance**
Synced the in-game-time to real-time. Every 1h real-time = 24h cycle in-game (hh:30 is always 12:00 in-game) before this commit worldTime is set to 14:27 when tfs is started.
I personally found the code that calculates worldLight to be a pain to read (could be that I'm just more of a noob than I thought) so I rewrote it.
Added a setWorldLight(level, color) function for any ot owners that want to play around with custom lighting. The function is only usable by setting "defaultWorldLight" in config to false so the custom and default don't overwrite each other.

Let me know if anything doesn't make sense/can be improved
Cheers

### CODE REVIEW

File: src/luascript.cpp [hunk 1]
```diff
@@ -1004,6 +1004,9 @@ void LuaScriptInterface::registerFunctions()
 	//getWorldLight()
 	lua_register(luaState, "getWorldLight", LuaScriptInterface::luaGetWorldLight);
 
+	//setWorldLight()
```
Comment (slavidodo) [inline src/luascript.cpp]: //setWorldLight(level, color)

File: config.lua.dist [hunk 2]
```diff
@@ -76,6 +76,7 @@ classicEquipmentSlots = false
 classicAttackSpeed = false
 showScriptsLogInConsole = true
 showOnlineStatusInCharlist = false
+defaultWorldLight = true
```
Comment (EPuncker) [inline config.lua.dist]: maybe add a comment -- telling what it does since I don't know it either 😛

**Comment by ranisalt** [comment]
Why not moving it straight to Lua? Just have a global interval event that calculates world time from system time and update it. Much cleaner and customizable :)

**Review by DSpeichert — APPROVED** [review]

**Review by MillhioreBT — APPROVED** [review]
Simply lovely. LGTM
tested and working...

**Pull request merged**